src/errors.ts

```diff
@@ -0,0 +1,8 @@
+import ErrorTimer from '@/ErrorTimer';
```

Comment:
We don't need this for libraries. A simple `src/errors.ts` is sufficient`.

Reply:
See `js-async-init` as an example. PK and EFS is actually unique in that they have more complex errors.

Reply:
This has been fixed.